languages/eo.lua

```diff
@@ -1,4 +1,87 @@
+-- Verkita originale por Fredrick BRENNAN (copypaste@kittens.ph) 2022-jare
+
 SILE.hyphenator.languages["eo"] = {exceptions={}}
 SILE.hyphenator.languages["eo"].patterns = {
-"a1", "b1", "c1", "ĉ1", "d1", "e1", "f1", "g1", "ĝ1", "h1", "ĥ1", "h1", "i1", "j1", "ĵ1", "k1", "l1", "m1", "n1", "o1", "p1", "r1", "s1", "ŝ1", "t1", "u1", "ŭ1", "v1", "z1",
+-- La fonto de la ŝablonoj ĉisubaj estas Sergio Pokrovskij (@pok49)'s GitHub-projekto «vortdivido».
+-- Ili uzendas LaTeX-permisile.
```

Comment:
I have not idea what this means. I don't mind the native language, but comments should IMHO always have an English version, for developers to understand what's at stakes.

Reply:
Oh, that's fine. I thought only E-o speakers would care, it's data sourcing information.